static-files/source/scss/style.scss

```diff
@@ -1,7 +1,6 @@
 @import "fonts";
 @import "vars";
 @import "mixins";
```

Comment:
The mixins file doesn't exist anymore, does it?

Reply:
Or rather: there doesn't seem to be anything in it anymore, so it should be removed.

Reply:
Both `vars.scss` and `mixins.scss` still exist as empty files. My thinking is that I (and probably others too) want those files for every new project and so I'd like to save the keystrokes and mouse clicks that it takes to create and import those files. To me much of the appeal of a tool like this is to be relieved of tedious manual labour